Project.toml

```
@@ -58,6 +59,7 @@ Octavian = "0.3.5"
 OffsetArrays = "1.3"
 P4est = "0.4"
 Polyester = "0.3.4, 0.5, 0.6, 0.7"
+PrecompileTools = "1"
```

```suggestion
PrecompileTools = "1.0.3"
```
Because of https://github.com/JuliaLang/PrecompileTools.jl/pull/15?

---

I requested an even newer version in 7354b2081

---

> I requested an even newer version in 7354b2081

That commit does not seem to exist?